modules/osdk-cli-reference-generate-packagemanifests.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-packagemanifests_{context}"]
+= packagemanifests
+
+Running `generate packagemanifests` subcommand is the first step to publishing your Operator to a catalog and/or deploying
+it with OLM. This command generates a set of manifests in a versioned directory and a package manifest file for
+your Operator. You must run `generate kustomize manifests` first to regenerate Kustomize bases
+consumed by this command.
+
+.`generate packagemanifests` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`--channel` (string)
+|The channel name for the generated package.
+
+|`--crds-dir` (string)
+|The root directory for custom resource definition (CRD) manifests.
+
+|`--default-channel`
+|Use the channel passed to `--channel` as the default channel of package manifest file.
+
+|`--deploy-dir` (string)
+|The root directory for Operator manifests such as deployments and RBAC, for example, `deploy`. This directory is different from that passed to `--input-dir`.
+
+|`--from-version` (string)
+|The semantic version of the Operator, from which it is being upgraded.
+
+|`-h`, `--help`
+|Help for `generate kustomize manifests`.
+
+|`--input-dir` (string)
+|The directory to read existing package manifests from. This directory is the parent of individual versioned package directories, and different from `--deploy-dir`.
+
+|`--kustomize-dir` (string)
+|The directory containing Kustomize bases and a `kustomization.yaml` for Operator-framework manifests. Default: `config/manifests`.
```

Comment:
Please revise "Operator-framework."  If it is a directory, then please revise to `operator-framework` according to the [quick markup reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) in the doc guidelines for file names or directory paths.

If the the purpose is to refer to the framework, then please use "Operator Framework" to match the capitalization on pages such as the [About the Operator SDK](https://docs.openshift.com/container-platform/4.6/operators/operator_sdk/osdk-about.html) page.